pkg/vfs/vfs.go

```diff
@@ -522,14 +522,18 @@ func (v *VFS) Release(ctx Context, ino Ino, fh uint64) {
 				}
 			}
 			locks := f.locks
-			owner := f.flockOwner
+			fowner := f.flockOwner
+			powner := f.ofdOwner
```

Comment:
record the owners in the access log?

Reply:
The owner come from previous setlk, we don't need to put it in accesslog.